### PR TITLE
Building Energy Standards Chiller Data Update

### PR DESCRIPTION
**Pull request by leijerry888**
Updates have been made to the Building Energy Standards Data repository, see [here](https://github.com/pnnl/building-energy-standards-data/commit/4b6a008c0bb9435586a2fd58fac06d0f26a26b69). This pull request was automatically generated.

### CODE REVIEW

File: lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb [hunk 1]
```diff
@@ -343,13 +343,13 @@ def hspf_to_cop_heating_with_fan(hspf)
   # @param eer [Double] Energy Efficiency Ratio (EER)
   # @param capacity_w [Double] the heating capacity at AHRI rating conditions, in W
   # @return [Double] Coefficient of Performance (COP)
-  def eer_to_cop(eer, capacity_w = nil)
+  def eer_to_cop_no_fan(eer, capacity_w = nil)
```
Comment (lymereJ) [inline lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb:346]: These functions were mislabeled. They did not correspond to a straight EER to COP conversion but rather excluded the effect of the fan when doing the conversion.

File: lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb [hunk 2]
```diff
@@ -343,13 +343,13 @@ def hspf_to_cop_heating_with_fan(hspf)
...
     if capacity_w.nil?
       # The PNNL Method.
       # r is the ratio of supply fan power to total equipment power at the rating condition,
       # assumed to be 0.12 for the reference buildings per PNNL.
       r = 0.12
-      cop = (eer / 3.413 + r) / (1 - r)
+      cop = (eer / OpenStudio.convert(1.0, 'W', 'Btu/h').get + r) / (1 - r)
```
Comment (lymereJ) [inline lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb:352]: Use the `convert` function of OS to get additional precision.

File: lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb [hunk 3]
```diff
@@ -382,6 +382,14 @@ def cop_to_eer(cop, capacity_w = nil)
     return eer
   end
 
+  # Convert from EER to COP
+  #
+  # @param cop [Double] Energy Efficiency Ratio (EER)
+  # @return [Double] Coefficient of Performance (COP)
+  def eer_to_cop(eer)
```
Comment (lymereJ) [inline lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb:389]: Add new function to do a straight EER to COP conversion.

File: lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb [hunk 4]
```diff
@@ -1138,6 +1138,7 @@ def model_set_central_preheat_coil_spm(model, thermal_zones, coil)
         setpoint_sch = tstat.heatingSetpointTemperatureSchedule
         setpoint_c = OpenstudioStandards::Schedules.schedule_get_design_day_min_max(setpoint_sch.get, 'winter')['max']
         next if setpoint_c.nil?
+
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb:1141]: Rubocop formatting.

**Comment by mdahlhausen** [comment]
Several tests are failing because the `kw_per_ton` variable is undefined [in this line](https://github.com/NREL/openstudio-standards/blob/97a42f698c492bad98d7a5f211ab94023d402617/lib/openstudio-standards/standards/Standards.ChillerElectricEIR.rb#L206C93-L206C103).

File: lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb (hunk 1 above)
Comment (lymereJ) [reply lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb:346]: @khaddad, @ckirney  - as per our discussion/email, see the changes in `lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb`.

File: lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlantLoop.rb [hunk 5]
```diff
@@ -120,11 +120,11 @@ def plant_loop_apply_prm_number_of_chillers(plant_loop, sizing_run_dir = nil)
     if cap_tons <= 300
       num_chillers = 1
       chiller_cooling_type = 'WaterCooled'
-      chiller_compressor_type = 'Rotary Screw and Scroll'
+      chiller_compressor_type = 'Rotary Screw'
     elsif cap_tons > 300 && cap_tons < 600
       num_chillers = 2
       chiller_cooling_type = 'WaterCooled'
-      chiller_compressor_type = 'Rotary Screw and Scroll'
+      chiller_compressor_type = 'Rotary Screw'
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.PlantLoop.rb:127]: @weilixu - There are a few PRM related changes.

**Comment by lymereJ** [comment]
@mdahlhausen - I think this is finally ready for review.

File: lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.ChillerElectricEIR.rb [hunk 6]
```diff
@@ -14,16 +14,17 @@ def chiller_electric_eir_apply_efficiency_and_curves(chiller_electric_eir)
 
     # Set the efficiency value
     cop = chiller_electric_eir_standard_minimum_full_load_efficiency(chiller_electric_eir)
-    kw_per_ton = cop_to_kw_per_ton(cop)
     if cop.nil?
```
Comment (mdahlhausen) [inline lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.ChillerElectricEIR.rb:17]: rubocop is suggesting guard clause format here:

unless cop
  # log stuff
  return false
end

**Review by mdahlhausen — COMMENTED** [review]
Thanks for the refactor of the chiller data. Two comments:
- Please update from the master branch. You will need to generate the regression models again, as those changed with recent updates to master.

- I understand the need to remove curves from the data .jsons and put them behind methods in standards instead. That may end up being the best approach, but I'm curious what other options you considered. In general, it'd be nice to have data elements in data files (perhaps a separate, static, chiller_curves.json), but could be (easily) convinced of your approach.

File: lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb (hunk 1 above)
Comment (ckirney) [reply lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb:346]: Thanks

**Comment by lymereJ** [comment]
@mdahlhausen - Thanks for the review!
- I updated the branch from master and added the guard clause
- I started looking into how the curves are handled a while back and quickly noticed that this aspect of openstudio-standards is probably due for a clean-up/refactor (for instance, we have some curves that hard-coded and not included in data files, and some that are in the data file that are somehow prototype specific), so I was hoping to handle it like this for now and once we've completed the code expansion of the database that we could perhaps dig into that and develop something better

**Pull request merged**